frontend/src/hooks/useProducts.ts

```diff
@@ -7,96 +7,87 @@ import { useLoading } from '../context/loading-context'
 
 const testProducts: Product[] = [
   {
-    id: 1,
-    name: 'Test Product',
-    price: 100,
-    description: 'This is a test product',
-    image: placeholderImage,
+    id: 11,
+    title: 'Innovative Product',
+    price_sgd: '250',
+    description: 'A groundbreaking product that changes the game',
+    image_url: placeholderImageVerticallyLong,
+    number_sold: 5,
+    category: 'Technology',
+    link: 'https://www.example.com',
   },
```

Comment:
You may want to remove this array altogether, since we are not using it anymore.